shared/chat/conversation/info-panel/index.desktop.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 import * as React from 'react'
-import {Box, Button, Checkbox, Divider, Icon} from '../../../common-adapters'
```

Comment:
I think we should see if we can merge this component into a shared one. If there isn't really a need for a split desktop/native we should evaluate if we can ditch them